torch/testing/_internal/hop_exportability_db.py

```diff
@@ -0,0 +1,103 @@
+# mypy: ignore-errors
+
+import functools
```

Comment:
1) can you merge this with https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/control_flow_opinfo_db.py ?
2) This should be a generic "hop_db", not just a HOP exportability db. We can reuse it for other tests, like autograd.